Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:latest
 
-ENV ISTIO_VERSION 1.9.8
+ENV ISTIO_VERSION 1.11.2
```

Comment:
1.11.3 is out, we should wait on updating these until closer to release. THere will likely be a security release between now and then anyway.